tracks/rust/exercises/bob/mentoring.md

```diff
@@ -126,13 +108,11 @@ If they put their constants inside the function:
 Note that it's common convention to put constants at the beginning of a file like I've done above. If it's a public constant, which is typical, we put it following our `use` lines, along with `struct`s and `enum`s. If it's a private constant, we put it at the beginning of the scope of the function. But, either way doesn't affect the performance or functionality!
 ```
 
-If they ask about `&str` having a static lifetime in comparison to using constants:
+If they ask about `&str` having a static lifetime:
 ```
-To quote from [Rust By Example](https://doc.rust-lang.org/rust-by-example/scope/lifetime/static_lifetime.html): "When `static_string` goes out of scope, the reference can no longer be used, but the data remains in the binary."
```

Comment:
This seems like great advice (with the link I mean). Can you explain what's wrong with it? Or is this part of the rest of the "change" for lifetimes?

Can we keep the quote + link in, but use your description?

Reply:
I'l add the link again.

I removed this section, because

1. The question in the heading is too vague.
2. The answer is in part incorrect.
3. The answer goes off-topic. I feel like it doesn't really answer the question(s).

Reply:
@SleeplessByte I added the link again.